app/assets/stylesheets/nav.scss

```diff
@@ -9,31 +9,59 @@ $button-color: #1E90FF;
   margin-left: 5%;
   list-style-type: none;
 }
-
+.top-bar {
+  background-color: #3f3f3f;
+}
+h3{
+  font-family: $font-sans;
+}
+h2{
+  color: white;
+}
+body{
+  background-color: #e9e9e9;
```

Comment:
Color literals like `#e9e9e9` should only be used in variable declarations; they should be referred to via variable everywhere else.